packages/gcab.rb

```diff
@@ -22,21 +22,13 @@ class Gcab < Package
   depends_on 'glibc' # R
   depends_on 'zlibpkg' # R
 
-  def self.prebuild
+  gnome
+
+  def self.patch
     system "sed -i 's,-fstack-protector-strong,-fno-stack-protector,' meson.build"
```

Comment:
This could probably be implemented with something like `pre_meson_options`, no?

Reply:
Honestly, it might not even need that line... I didn't check for a package update to see if it now builds without it. I just converted it to our current format.

Reply:
Didn't need the line.  Updated and rebuilt.